src/adv_diff/BrinkmanAdvDiffSemiImplicitHierarchyIntegrator.cpp

```diff
@@ -535,7 +547,7 @@ BrinkmanAdvDiffSemiImplicitHierarchyIntegrator::integrateHierarchy(const double
             // set 1.0/dt - (1.0-K)*Cb in rhs_op_spec, which is applied to Q later
             d_hier_cc_data_ops->scale(Cb_rhs_scratch_idx, -(1.0 - K), Cb_current_idx);
             // In the case of Poisson applications zero out the temporal contribution.
-            if (d_brinkman_time_independent)
+            if (d_brinkman_time_independent[Q_var])
             {
```

Comment:
use d_Q_time_independent instead.

Reply:
BTW, can we do this check for all Q_vars here instead of just those marked as Brinkman one? That would be more consistent.

Reply:
And in that case `setBrinkmanVariableTimeIndependent()` should be named `setTransportQuantityTimeIndependent()`.